content/developer/reference/cli.rst

```diff
@@ -695,6 +695,11 @@ interaction with the :ref:`orm <reference/orm>` and its functionalities.
       By default, the shell is running in transaction mode. This means that any change made to the
       database is rolled back when exiting the shell. To commit changes, use `env.cr.commit()`.
 
+
+.. option:: --shell-file <init_script.py>
+   Specify a python script to be run after the start of the shell.
+   Overrides the environment variable PYTHONSTARTUP.
```

Comment:
Single line breaks are not taken into account in RST. You need to add a blank line after `option`, otherwise it will look like [that](https://runbot117.odoo.com/runbot/static/build/70270290-master/logs/build/html/developer/reference/cli.html#cmdoption-odoo-bin-shell-file).

```suggestion
.. option:: --shell-file <init_script.py>

   Specify a python script to be run after the start of the shell.
   Overrides the environment variable PYTHONSTARTUP.
```

Reply:
For the same reason, you might want to *not* return to a new line at the end of your sentence if you don't intend to have a line return in the rendered HTML:
```suggestion
.. option:: --shell-file <init_script.py>
   Specify a python script to be run after the start of the shell. Overrides the environment variable PYTHONSTARTUP.
```